/skeleton_markers:
  ros__parameters:
    # Marker color rgba
    color:
    - 0.0
    - 1.0
    - 0.0
    - 1.0
    # The fixed reference frame
    fixed_frame: camera_depth_frame
    # Marker id
    id: 0
    # Duration of markers in RViz; 0 is forever
    lifetime: 0
    # Marker namespace
    ns: skeleton_markers
    # Update rate
    rate: 30
    # Height and width of markers in meters
    scale: 0.07
    # List of frames
    skeleton_frames:
    - head
    - neck
    - torso
    - left_shoulder
    - left_elbow
    - left_hand
    - left_hip
    - left_knee
    - left_foot
    - right_shoulder
    - right_elbow
    - right_hand
    - right_hip
    - right_knee
    - right_foot
    # qos
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false
